### DISCUSSION TITLE
rockchip/repo is not accessible

### DISCUSSION THREAD

**Discussion by vanthihan** [community/community#108616] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,

I am getting started with rk3399 and following [this development guideline](http://opensource.rock-chips.com/images/archive/f/f9/20170712094221%21RK3399_Linux_Debain_V1.1_Development_Guide170620.pdf) but in the command:
`repo init --repo-url=https://github.com/rockchip-linux/repo -u https://github.com/rockchip-linux/manifests -b master`

I faced below errors:

<img width="914" alt="Screenshot 2024-02-17 211937" src="https://github.com/community/community/assets/90036635/ff64be6b-1509-48dd-8f53-820c4bc5c719">

Do I have to register for access or any instruction to follow? Please suggest.

Thanks!

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @vanthihan! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by JakeB999** [comment]
Hi Vanthihan, did you work out an answer for this? I am having the same problem. Thanks. 

> ### Select Topic Area
> 
> Question
> ### Body
> 
> Hi,
> 
> I am getting started with rk3399 and following [this development guideline](http://opensource.rock-chips.com/images/archive/f/f9/20170712094221%21RK3399_Linux_Debain_V1.1_Development_Guide170620.pdf) but in the command: `repo init --repo-url=https://github.com/rockchip-linux/repo -u https://github.com/rockchip-linux/manifests -b master`
> 
> I faced below errors:
> <img alt="Screenshot 2024-02-17 211937" width="914" src="https://private-user-images.githubusercontent.com/90036635/305645872-ff64be6b-1509-48dd-8f53-820c4bc5c719.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAwODc1NDIsIm5iZiI6MTcyMDA4NzI0MiwicGF0aCI6Ii85MDAzNjYzNS8zMDU2NDU4NzItZmY2NGJlNmItMTUwOS00OGRkLThmNTMtODIwYzRiYzVjNzE5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA0VDEwMDA0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2YjU5MmQwOTUwNzIyOTM3NTk4OGJkOWQ2NTA3ODY2Y2VhYmI0Y2I2NjBlZWMzNmU1NjZhMWEwZTkyNmEyNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ForHPKn3fJUAL64CK0kQ0Yp7SaFsx6x16_d8FcVfBMg">
> 
> Do I have to register for access or any instruction to follow? Please suggest.
> 
> Thanks!

**Reply by vanthihan** [reply]
Hi Jake,

I emailed to rockchip and they response that I have to buy some chip or product related to access this sdk,
But I found other option with Toybrick EVB which also use with RK3399 at https://t.rock-chips.com/en/wiki.php?filename=Downloads/Downloads
